UnityProject/Packages/packages-lock.json

```diff
@@ -48,7 +48,7 @@
       "dependencies": {}
     },
     "com.unity.2d.tilemap.extras": {
-      "version": "2.2.4",
+      "version": "2.2.5",
```

Comment:
I think you shouldn't push this file

Reply:
Yeah, probably. I'll try and do it now.

Reply:
Okie dokie, fixed in the commit 7362e4426c635b30629ec37d210a48a3f654fb37...